openpype/hosts/maya/plugins/create/create_animation_pointcache.py

```diff
@@ -0,0 +1,287 @@
+            items=["ogawa", "HDF"],
+            disabled=True if "dataFormat" not in abc_args_overrides else False,
+        )
+    )
+
+    abc_defs.append(
+        NumberDef(
+            "preRollStartFrame",
+            label="Start frame for preroll",
+            default=getattr(cls, "preRollStartFrame", None),
+            tooltip=(
+                "The frame to start scene evaluation at. This is used to set"
+                " the starting frame for time dependent translations and can"
+                " be used to evaluate run-up that isn't actually translated."
+            ),
+            disabled=True
+            if "preRollStartFrame" not in abc_args_overrides
+            else False,
+        )
+    )
+
+    return abc_defs
+
+
+def _ensure_defaults(cls, instance_data):
+    """Ensure we get default values when an attribute is not overrideable.
+
+    In instances where an attribute used to be modifiable, and then was locked
+    again, we want to make sure that we pass the default (what's on the
+    settings) instead of any value that might have been stored in the scene
+    when the attribute was modifiable.
+    """
+    abc_args_overrides = getattr(cls, "abc_args_overrides", [])
+    creator_attr = instance_data["creator_attributes"]
+    attr_default = getattr(cls, "attr", "")
+
+    if "attr" not in abc_args_overrides:
+        creator_attr["attr"] = attr_default
+
+    if "attrPrefix" not in abc_args_overrides:
+        creator_attr["attrPrefix"] = getattr(cls, "attrPrefix", "")
+
+    if "dataFormat" not in abc_args_overrides:
+        creator_attr["dataFormat"] = getattr(cls, "dataFormat", "")
+
+    if "preRollStartFrame" not in abc_args_overrides:
+        creator_attr["preRollStartFrame"] = getattr(
+            cls, "preRollStartFrame", ""
+        )
+
+    (
+        abc_boolean_defaults,
+        abc_boolean_overrides,
+    ) = _get_alembic_boolean_arguments(cls)
+
+    creator_attr["abcDefaultExportBooleanArguments"] = abc_boolean_defaults
+
+    creator_attr["abcExportBooleanArguments"] = [
+        arg for arg in creator_attr["abcExportBooleanArguments"]
+        if arg not in abc_boolean_overrides
+    ]
+
+    return instance_data
+
+
+class CreateAnimation(plugin.MayaHiddenCreator):
+    """Animation output for character rigs
+
+    We hide the animation creator from the UI since the creation of it is
+    automated upon loading a rig. There's an inventory action to recreate it
+    for loaded rigs if by chance someone deleted the animation instance.
+    """
+
+    identifier = "io.openpype.creators.maya.animation"
+    name = "animationDefault"
+    label = "Animation"
+    family = "animation"
+    icon = "male"
+
+    write_color_sets = False
+    write_face_sets = False
+    include_parent_hierarchy = False
+    include_user_defined_attributes = False
+
+    def collect_instances(self):
+        cached_subsets = self.collection_shared_data["maya_cached_subsets"]
```

Comment:
beware, `maya_cached_subsets` might not exist at this time. It is run in `collect_instances()` in **MayaHiddenCreator** but here, you either need to call `super()` or run it manually.

```suggestion
        cached_subsets = self.collection_shared_data["maya_cached_subsets"]
```